catalyst_voices/lib/widgets/buttons/voices_keyboard_key_button.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:catalyst_voices/widgets/tooltips/voices_plain_tooltip.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:flutter/material.dart';
+
+/// Widget that is meant to be used as singular keyboard key indication.
+///
+/// Common use case is to show list of such keys that together means
+/// shortcut of some sorts.
+///
+/// See:
+///   * [VoicesPlainTooltip] as good starting points for used of this button.
+class VoicesKeyboardKeyButton extends StatelessWidget {
+  /// Usually [Icon], [CatalystSvgIcon] or [Text].
+  final Widget child;
+
+  const VoicesKeyboardKeyButton({
+    super.key,
+    required this.child,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    final iconTheme = IconThemeData(
+      size: 14,
+      color: theme.colors.iconsForeground,
+    );
+
+    final textStyle = TextStyle(
+      fontSize: 14,
+      fontWeight: FontWeight.w700,
```

Comment:
Can we find one of text styles from text theme or is this text style entirely custom?